docs/guides/deployment/kubernetes.md

```diff
@@ -69,15 +76,118 @@ Besides a cluster, the other critical resource we'll need is a container registr
 
 ## Create a work pool
 
-Now let's switch over to Prefect Cloud, where we'll create a new work pool. Hit the plus button on the work pools page and choose Kubernetes from the list of options. On the next page, set Finished Job TTL to 60 so that completed flow runs are cleaned up, and set Pod Watch Timeout Seconds to 300, especially if you are using a __serverless__ type node pool since these tend to have longer startup times. You may also want to set a custom namespace, such as `prefect`. Generally you should leave the cluster config blank as the worker will already be provisioned with appropriate access and permissions. Finally, leave the image field blank as we'll override that in each deployment.
+Now let's switch over to Prefect Cloud, where we'll create a new work pool. 
+Hit the plus button on the work pools page and choose Kubernetes from the list of options. 
+On the next page, set Finished Job TTL to 60 so that completed flow runs are cleaned up, and set Pod Watch Timeout Seconds to 300, especially if you are using a __serverless__ type node pool since these tend to have longer startup times. 
+You may also want to set a custom namespace, such as `prefect`. 
+Generally you should leave the cluster config blank as the worker will already be provisioned with appropriate access and permissions. 
+Finally, leave the image field blank as we'll override that in each deployment.
 
 ## Deploy a worker using Helm
 
-With our cluster and work pool created, it's time to deploy a worker, which will take our flows and run them as Kubernetes jobs. The best way to deploy a worker is using Helm. Follow [this guide](/guides/deployment/helm-worker/) and then come back here.
+With our cluster and work pool created, it's time to deploy a worker, which will take our flows and run them as Kubernetes jobs. 
+The best way to deploy a worker is using the [Prefect Helm Chart](https://github.com/PrefectHQ/prefect-helm/tree/main/charts/prefect-worker). 
+
+### Step 1: Add Prefect Helm repository
+
+Add the Prefect Helm repository to your Helm client:
+
+<div class="terminal">
```

Comment:
Do we need the `<div class="terminal">` in addition to ```bash?

Reply:
Ah, looking more closely at the style guide, the current guidance is  "Wrap terminal output in `<div class=terminal> </div>" - this is input, so it appears it should not be wrapped. I'll fix it. Thank you for raising.